src/library/Uno.Material/Controls/Chips/ChipGroup.cs

```diff
@@ -127,11 +136,20 @@ private void OnSelectedItemsChanged(DependencyPropertyChangedEventArgs e)
 		{
 			if (_isSynchronizingSelection || _isUpdatingSelection) return;
 
+			if (!_needsToSynchronizeInitialSelectionForSelectedItems && ItemsPanelRoot == null && e.NewValue != null)
+			{
+				_needsToSynchronizeInitialSelectionForSelectedItems = true;
+				return;
+			}
+
 			if (SelectionMode == ChipSelectionMode.Multiple)
 			{
-				var selectedContainers = SelectedItems != null
-					? GetItemContainers().Where(x => SelectedItems.Contains(x)).ToArray()
-					: null;
+				var selectedContainers = SelectedItems
+				   .Cast<object>()
```

Comment:
SelectedItems can be null here.
```suggestion
				   ?.Cast<object>()
```